db/seeds.rb

```diff
@@ -11,11 +11,13 @@ def create_users()
     User.create( 
       first_name:   Faker::Name.first_name, 
       last_name:    Faker::Name.last_name,
-      linkedin_id:  Faker::Lorem.characters(10),
+      description:  Faker::Lorem.paragraph,
       contact_url:  Faker::Internet.url,
-      visible:      Faker::Boolean.boolean(0.8),
-      role:         :user,
-      description:  Faker::Lorem.paragraph
+      provider:     :linkedin,
```

Comment:
I don't know how the works exactly, wouldn't we want this to just be "linkedin" as a string?

Reply:
The :linkedin symbol gets converted to a string when the record is created